docs/conf.py

```diff
@@ -61,7 +61,7 @@
 
 # autodoc/autosummary flags
 autoclass_content = "both"
-autodoc_default_options = {"members": True, "inherited-members": True}
```

Comment:
I'm curious about this change. What inherited members were causing problems? IIRC, there's a few methods defined in the base class for jobs that we want to make sure are documented.

Reply:
Are there plans to restore the missing docs, such as `reservation` and `job_timeout_ms` on the `*JobConfig` classes?